RetroBar/Languages/English.xaml

```diff
@@ -65,6 +65,7 @@
         <s:String>Open Modern calendar</s:String>
         <s:String>Open Notification Center</s:String>
     </x:Array>
+    <s:String x:Key="auto_hide_transparent">Hide _taskbar edge with auto-hide</s:String>
```

Comment:
"Taskbar edge" seems a bit friendlier than my alternative "sizing bar", which is used in the original theming terminology. Still, I'm not quite sure why it should be used for that reason alone.

Reply:
Perhaps "sizing bar line" would make more sense.

Reply:
How about "Hide _taskbar line when hidden with auto-hide"?

Reply:
I like that better!